package.json

```diff
@@ -114,7 +114,7 @@
     "ember-cli-terser": "^4.0.2",
     "ember-cli-typescript": "^5.2.1",
     "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-intl": "5.7.2",
```

Comment:
do you need 6.1? afaik we had some issues with it @elpoelma if you could weigh in

Reply:
Ah, I saw [this commit](https://github.com/lblod/ember-rdfa-editor/pull/1122) moving to 6.1 and assumed this had been done on this branch to avoid adding another breaking release and instead combining it with the rdfa work. I see now that the PR was actually to master and then it was reverted again later. The confusion was that the commit moving back to 5.7 wasn't in `feat/rdfa-links-ui`, so I got unresolved peer dep errors.

I'll merge master into that branch and then rebase this without that commit.